DXMainClient/PreStartup.cs

```diff
@@ -146,7 +146,7 @@ public static void Initialize(StartupParams parameters)
 
                     Logger.Log("Translation stub generation feature is now enabled. The stub file will be written when the client exits.");
 
-                    // Load all literal strings
+                    // Lookup all comptime available strings
```

Comment:
typo

Reply:
Where?

Reply:
compile-time

Reply:
comptime is a common shortening for that